.eslintignore

```diff
@@ -0,0 +1,3 @@
+dist
```

Comment:
Can you please add the `coverage` folder to the default ignore list?

(an existing report threw all kinds of errors when I ran locally, and it's a pretty reasonable thing for someone to have locally)

I did a quick check of gitignore and otherwise this looks good.